include/swupd-build-variant.h

```diff
@@ -11,15 +11,19 @@
 #ifdef SWUPD_WITH_BSDTAR
 #define TAR_COMMAND "bsdtar"
 #define TAR_XATTR_ARGS ""
+#define TAR_XATTR_ARGS_STRLIST
 #else
 #define TAR_COMMAND "tar"
 #define TAR_XATTR_ARGS "--xattrs --xattrs-include='*'"
+#define TAR_XATTR_ARGS_STRLIST "--xattrs", "--xattrs-include='*'",
 #endif
 
 #ifdef SWUPD_WITH_SELINUX
 #define TAR_PERM_ATTR_ARGS "--preserve-permissions --selinux " TAR_XATTR_ARGS
+#define TAR_PERM_ATTR_ARGS_STRLIST TAR_XATTR_ARGS_STRLIST "--preserve-permissions", "--selinux"
 #else /* SWUPD_WITHOUT_SELINUX */
 #define TAR_PERM_ATTR_ARGS "--preserve-permissions " TAR_XATTR_ARGS
+#define TAR_PERM_ATTR_ARGS_STRLIST TAR_XATTR_ARGS_STRLIST "--preserve-permissions"
```

Comment:
@jrguzman-intel Perhaps you can reference `TAR_XATTR_ARGS_STRLIST` at the end of the definition instead, to not need the trailing comma at all?

```
#define TAR_PERM_ATTR_ARGS_STRLIST  "--preserve-permissions", TAR_XATTR_ARGS_STRLIST
```